tuna/import_db.py

```diff
@@ -260,8 +213,7 @@ def main():
   """main"""
   args = parse_args()
   args.table_cfg = ConvolutionConfig
-  args.table_perf_cfg = ConvPerfConfig
-  args.table_perf_db = ConvPerfDB
+  args.table_perf_db = ConvolutionFindDB
```

Comment:
can you please add the dbt structure to this script? initialize and then use its tables instead of importing specific miopen tables here.